src/components/InputImage/InputImage.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState, useRef } from 'react';
+
+import scss from './InputImage.module.scss';
+
+function InputImage() {
+  const [imageSelected, setImageSelected] = useState<File | null>(null);
+  const fileInputRef = useRef<HTMLInputElement>(null);
+
+  const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const selectedImage = event.target.files?.[0];
+    if (selectedImage) {
+      setImageSelected(selectedImage);
+      // Mostrando informações da imagem durante o desenvolvimento
+      // eslint-disable-next-line no-console
+      console.log('imagem', selectedImage);
+    }
+  };
+
+  const handleImageIconClick = () => {
```

Comment:
Vc deve pegar a imagem a partir do evento que viria:

```suggestion
  const handleImageIconClick = (event) => {
```